### PR TITLE
Release-1.29 kubeadm: add note about a bug in the PublicKeysECDSA feature gate

### PR DESCRIPTION
**Pull request by SequoiaGod**
Add note about that in the FG section of the kubeadm init doc.
xref https://github.com/kubernetes/website/issues/46799

### CODE REVIEW

**Review by neolit123 — COMMENTED** [review]
/lgtm
/sig cluster-lifecycle
/triage accepted
/assign @sftim
/label tide/merge-method-squash

File: api-ref-generator
Comment (dipesh-rawat) [inline api-ref-generator:1]: Thanks for the fix! 
Just a quick question: Is the update to this git submodule an intentional part of the changes, or might it have been included by mistake?

Comment (neolit123) [reply api-ref-generator:1]: please remove this change @SequoiaGod

Comment (SequoiaGod) [reply api-ref-generator:1]: @neolit123  @dipesh-rawat That's pretty weired. The rest of code has been auto-generated , I didn't realise it when I made this PR. I will close this PR as git is going crazy and it still auto-add those changes to my commit. New PR: https://github.com/kubernetes/website/pull/46898

**Pull request closed without merging**